clippy_lints/src/operators/arithmetic_side_effects.rs

```diff
@@ -98,8 +98,15 @@ impl ArithmeticSideEffects {
     }
 
     /// If `expr` is not a literal integer like `1`, returns `None`.
+    ///
+    /// It is worth nothing that the returned number can or can not be negative.
```

Comment:
Would you mind explaining what you meant with the new comment?

If you mean 'noting', I'd suggest the following doc comment for the function:

```suggestion
    /// Returns the absolute value of the expression, if this is an integer literal.
```

Reply:
Done